src/components/RoundForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <h2>Input Rounds</h2>
+    <p>{{ msg }}</p>
+    
+    <input v-model="date" type="text" size="35" placeholder="Enter date, yyyy-mm-dd"><br><br>
+    
+    <select v-model="club" @change="changeClub">
+      <option :value="{}" default selected disabled>Select club</option>
+      <option v-for="clubOpt in clubOptions" :key="clubOpt.name"
+      :value="clubOpt">
+        {{ clubOpt.name }}
+      </option>
+    </select><br><br>
+
+    <table>
+      <tr v-for="(course, i) in courses" :key="course.name">
+        <td>{{ course.name }}</td>
+        <td>
+          <button @click="removeCourse(i)">Remove</button>
+        </td>
+      </tr>
+      <tr v-show="canAddCourses">
+        <td>
+          <select v-model="newCourse">
+            <option :value="{}" default selected disabled>Select course</option>
+            <option v-for="course in newCourseOptions" :key="course.name"
+            :value="course">
+              {{ course.name }}
+            </option>
+          </select>
+        </td>
+        <td>
+          <button @click="addCourse"
+          :disabled="Object.keys(newCourse).length == 0">
+            Add
+          </button>
+        </td>
+      </tr>
+    </table><br>
+
+    <div v-for="(course, courseIndex) in courses" :key="course.name">
+      <h4>Course - {{ course.name }} </h4>
+      
+      <select v-model="course.tees">
+        <option value="" default selected disabled>Select tees</option>
+        <option v-for="tees in course.teesOptions" :key="tees">
+          {{ tees }}
+        </option>
+      </select><br><br>
+
+      <input v-model.number="course.numHoles" number type="number" 
+      min="1" max="18" size="35" placeholder="# holes" 
+      @change="changeNumHoles(courseIndex)"><br><br>
+      
+      <table border="1" cellpadding="4">
+        <tr>
+          <th>Hole</th>
+          <th v-for="num in course.numHoles" :key="num">{{ num }}</th>
+        </tr>
+        <tr v-for="(score, scoreIndex) in course.scores"  :key="score.player">
+          <td>
+            {{ score.player }}
+            <button @click="removeScore(scoreIndex, courseIndex)">
+              Remove
+            </button>
+          </td>
+          <td v-for="num in course.numHoles" :key="num">
+            <input v-model.number="score.strokes[num - 1]" number 
+            type="number" style="width: 35px;" min="0" max="20">
+          </td>
+        </tr>
+        <tr v-show="canAddScores">
```

Comment:
`v-show` instead of `v-if` since this gets toggled often (every time a 4th player is added, most rounds have 4 players, ...)